R/setup_parts.R

```diff
@@ -153,6 +153,15 @@ get_region_setup <- function(){
                  "asia-northeast1",
                  "europe-west1",
                  "us-east1")
+    choices <- c(choices,
```

Comment:
This looks like a list to maintain over time, perhaps there is an API call to get the regions to make this easier.